osquery/tables/system/linux/python_packages.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#include <string>
+#include <fstream>
+
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+#include <boost/algorithm/string/split.hpp>
+#include <boost/algorithm/string/trim.hpp>
+
+namespace osquery {
+namespace tables {
+
+const std::string python_path = "/usr/local/lib/python2.7/dist-packages/";
+
+void genPackage(std::string path, Row &r) {
+  std::ifstream fd (path, std::ios::in | std::ios::binary);
```

Comment:
We have a couple of `readFile` functions [here](https://github.com/facebook/osquery/blob/master/include/osquery/filesystem.h#L56) that you could use instead of external libraries, they should make things a bit easier :)